src/i18n.ts

```diff
@@ -1,23 +1,25 @@
-import { i18n } from '@lingui/core';
-import { detect, fromStorage } from '@lingui/detect-locale';
-import { en, de, es, fr, ja, ko, ru, uk, zh } from 'make-plural/plurals';
-import { messages as enMessages } from './locales/en/messages';
-import { messages as deMessages } from './locales/de/messages';
-import { messages as esMessages } from './locales/es/messages';
-import { messages as frMessages } from './locales/fr/messages';
-import { messages as jaMessages } from './locales/ja/messages';
-import { messages as koMessages } from './locales/ko/messages';
-import { messages as ruMessages } from './locales/ru/messages';
-import { messages as ukMessages } from './locales/uk/messages';
-import { messages as zhMessages } from './locales/zh/messages';
-import { messages as tokMessages } from './locales/jbo-tok/messages'
-
-const storageKey = 'uiLanguage';
+import { i18n } from "@lingui/core"
+import { detect, fromStorage } from "@lingui/detect-locale"
+import { en, de, es, fa, fr, ja, ko, ru, uk, zh } from "make-plural/plurals"
+import { messages as enMessages } from "./locales/en/messages"
+import { messages as deMessages } from "./locales/de/messages"
+import { messages as esMessages } from "./locales/es/messages"
+import { messages as faMessages } from "./locales/fa/messages"
+import { messages as frMessages } from "./locales/fr/messages"
+import { messages as jaMessages } from "./locales/ja/messages"
+import { messages as koMessages } from "./locales/ko/messages"
+import { messages as ruMessages } from "./locales/ru/messages"
+import { messages as ukMessages } from "./locales/uk/messages"
+import { messages as zhMessages } from "./locales/zh/messages"
+import { messages as tokMessages } from "./locales/jbo-tok/messages"
+
+const storageKey = "uiLanguage"
```

Comment:
Let's stick to the ending semicolons since that is the convention followed in the code base.

Reply:
> Is there a formatter config in this repo I have overlooked?

Not yet. I created #160 to fix that.